### DISCUSSION TITLE
Bad audio Quality and Scratching

### DISCUSSION THREAD

**Discussion by duestii** [DistroAV/DistroAV#804] [Q&A - Support]
Hello,

I set out to transmit a stream via ndi that runs very well and fluently, only the sound doesn't quite play along and it is "consumed" or generally in poor quality does anyone here know about ndi? Or has an idea what it could be that the sound is so "bad" is transmitted?
Sound and video are transmitted via ndi source i.e. ndi plug in for obs.

The NDI source streams the scene from the gaming PC to the streaming PC via the local network.

Both PCs are plugged into a GBit switch. Image liquid only sound is scratchy.

NDI vers is 5.5, i.e. the latest and obs vers is also the latest 28.1.2

gaming pc:
Ryzen 7 5800x
AMD rx6800xt
32GB RAM

Streaming PC/Laptop:
Lenovo IdeaPad 3
I5 10120U 1.6GHz
8GB RAM

Kind regards

**Comment by rave2n** [comment]
Set all sent audio to left on the streaming PC, mono, no stereo. Only solution I've found.

**Comment by Trouffman** [comment]
Hey @rave2n THis is related to a known bug on this brnach trying to be fixed in : https://github.com/Palakis/obs-ndi/pull/755

In the mean time, this has been solved, in a fork of this plugin :
https://github.com/DDRBoxman/obs-ndi/releases/tag/4.10.1
